Changes

```diff
@@ -49,6 +49,11 @@ Working version
 - #13354: Use C99 flexible array member syntax everywhere.
   (Antonin Décimo, review by Miod Vallat, Gabriel Scherer, and Xavier Leroy)
 
+- #13582: Enable software prefetching support for ARM64 on Linux and macOS.
```

Comment:
The logic to enable prefetching is based upon the compiler capabilities, not the target operating system, so that sentence is wrong.
```suggestion
- #13582: Enable software prefetching support for ARM64.
```

Reply:
Quite correct :-) 
More accurately I should have said I've only tested on those platforms. I don't have access to any ARM64 *BSD platforms to test the changes there.